### PR TITLE
[ADD] support mail in config view

### PR DESCRIPTION
**Pull request by ventilooo**
screenshot-2018-1-9 general settings - odoo 1

### CODE REVIEW

**Review by joaoalf — CHANGES_REQUESTED** [review]
@ventilooo I don't see where in the code we tell the system to use the email in ir.config_parameter.

The file names don't correspond to the models.

**Comment by ventilooo** [comment]
@joaoalf 
If you check the other module in odoo, like [this one](https://github.com/odoo/odoo/blob/10.0/addons/mail/models/res_config.py). The file name is `res_config.py` but the Class is `BaseConfiguration`.

For the code about `ir.config_parameter` => `standard_plus_issue/models/res_config.py`

**Comment by ventilooo** [comment]
@joaoalf fixed according to your review !

**Review by rimbendhaou — APPROVED** [review]
It's ok

File: standard_plus_issue/README.rst [hunk 1]
```diff
@@ -4,7 +4,7 @@ This module allows users to submit their Odoo issues to your technical support
 services, via an icon ('/static/img/sfl_logo.png') in the navigation bar.
 The user describes the issue and join screenshots. The request is then emailed
 to you.
-Replace 'support@support.com' in '/data/ir_property_support_email.xml' with
+Replace 'support@support.com' in '/data/ir_config_parameter.xml' with
```
Comment (joaoalf) [inline standard_plus_issue/README.rst]: @ventilooo Ça ne fait pas du sens.

**Review by joaoalf — CHANGES_REQUESTED** [review]
@ventilooo Voir mon commentaire pour la doc. Sinon, rebase + traductions.

File: standard_plus_issue/models/standard_plus_issue.py [hunk 2]
```diff
@@ -161,7 +156,9 @@ def action_set_submitted(
         self.send_by_email(self.prepare_mail_attachments(), template_ref)
         self.state = 'submitted'
         self.message_post(
-            body=_('Email sent to ') + self.property_support_email,
+            body=_('Email sent to ') + (
+                self.env['base.config.settings'].get_default_support_email(
```
Comment (joaoalf) [inline standard_plus_issue/models/standard_plus_issue.py]: @ventilooo ??

**Review by joaoalf — CHANGES_REQUESTED** [review]
@ventilooo You're not picking the address from the good place.

**Comment by ventilooo** [comment]
@joaoalf review ;)

**Review by joaoalf — APPROVED** [review]

**Pull request merged**